src/commands.ts

```diff
@@ -138,6 +138,38 @@ export function registerCommands(
 		),
 	);
 
+	context.subscriptions.push(
+		vscode.commands.registerCommand(
+			'pr.openAllChanges',
+			async () => {
+				const activePullRequestsWithRootUri = reposManager.folderManagers
+					.filter(folderManager => folderManager.activePullRequest)
+					.map(folderManager => {
+						return (({ activePr: folderManager.activePullRequest!, rootUri: folderManager.repository.rootUri }));
+					});
+
+				const activePullRequestWithRootUri = activePullRequestsWithRootUri.length >= 1
+					? (
+						await chooseItem(
+							activePullRequestsWithRootUri,
+							itemValue => itemValue.activePr.html_url,
+						)
+					)
+					: activePullRequestsWithRootUri[0];
+
+				if (!activePullRequestWithRootUri) {
+					return;
+				}
+
+				const { activePr, rootUri } = activePullRequestWithRootUri;
+				(await activePr.getFileChangesInfo())
```

Comment:
`getFileChangesInfo` can be expensive. Instead of of calling `getFileChangesInfo` again (it's called as part of setting up the compare changes tree), you should be able to use the folder manager to get the review manager using  
https://github.com/joshuaobrien/vscode-pull-request-github/blob/a721a36d310522bcda872915886d0f48e6a9657f/src/view/reviewManager.ts#L976-L977
then, there's a change on `main` that adds a `ReviewModel` to the `ReviewManager`:
https://github.com/microsoft/vscode-pull-request-github/blob/6711e556f502bbd13cc9b9e9de8475eea746b488/src/view/reviewManager.ts#L40-L41 
if you add a getter for `_reviewModel` in `ReviewManager`, then you can the `localFileChanges` from the `ReviewModel` and call the `openFileCommand` method on each of the `localFileChanges`.